mujoco_warp/_src/collision_gjk.py

```diff
@@ -27,7 +27,21 @@
 FLOAT_MAX = 1e30
 MJ_MINVAL2 = MJ_MINVAL * MJ_MINVAL
 
+FACE_TOL = 0.99999872
```

Comment:
how were these magic numbers derived?  can we get away without them?

if they are needed, can you add some notes in case someone ever has to go in and adjust them, they will know how to do it?